lib/bundler.js

```diff
@@ -38,8 +39,9 @@ module.exports = class ElementBundler extends Plugin {
 		return bundler.generateManifest([entry])
 			.then((manifest) => bundler.bundle(manifest)).then((result) => {
 				const html = parse5.serialize(result.documents.get(entry).ast);
+				const content = this.autoprefixStyles ? htmlAutoprefixer.process(html) : html;
```

Comment:
It would be even nicer if this could work seamlessly with that addon without any extra external dependency using the config of the main app.

This feature gets extra points 💃

Reply:
I think that a `.browserslistrc` file in the app root or a `browserslist` key in `package.json` should work.

Reply:
On Ember, we use `config/targets.js` to specify browser target versions:

https://guides.emberjs.com/release/configuring-ember/build-targets/